### DISCUSSION TITLE
Ideas for December

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#3057] [Community Call]
Many things going on in December (we will start a thread for that below), what else could be fun to do? Let's discuss 👇

**Comment by eddiejaoude** [comment]
What things like 24 Pull Requests are going on in December? 👇

**Reply by suitupstan** [reply]
- EddieHub Weekly Community Call + Eddie's Twitter Space 😜

**Reply by 0xMukesh** [reply]
Advent of code 👀

**Reply by 0xMukesh** [reply]
[KWoC](https://kwoc.kossiitkgp.org/)
**Kharagpur Winter of Code** is a 5-week long online programme for students who are new to open source software development. The programme not only helps students to get involved in open-source, but also prepares them for many open-source summer programmes; Google Summer of Code being one of them. The program is hosted by Kharagpur Open Source Society an independent student group at IIT Kharagpur, but is open to students of any university

**Comment by suitupstan** [comment]
Add streamyard for livestream (streamyard supports teams now)

**Comment by eddiejaoude** [comment]
Tron community game similar to BattleSnakes - idea by Vyvy-vi

**Comment by suitupstan** [comment]
Create a video explaining GSOC and create a video explaining GitHub Externships

**Reply by Vyvy-vi** [reply]
I think Eddie did a stream with kunal on this some time ago...

**Comment by suitupstan** [comment]
Nextjs LinkFree + Awesome GitHub Profiles website in Nextjs

**Reply by Vyvy-vi** [reply]
I'm wondering, couldn't we use a static site generator for Awesome Github Profiles?

**Comment by suitupstan** [comment]
Centered Call v2

**Comment by suitupstan** [comment]
Pre-LHD Build 2022 live streams

- What is LHD Build?
- Way to team up for LHD Build
- (My favourite) way to have fun while LHD Build (hangouts and sutff)

**Comment by schmelto** [comment]
Add a campfire effect for our websites on 24th 😂😂

**Reply by Vyvy-vi** [reply]
:D that would be awesome

**Comment by Pradumnasaraf** [comment]
Create a video on "Do's and Don't while creating a PR"

**Reply by schmelto** [reply]
https://youtu.be/ExFCwFPTsE0

**Reply by naomi-lgbt** [reply]
@Kira272921 Please do not post irrelevant links. This is not behaviour that belongs in our community. 🙂

**Reply by 0xMukesh** [reply]
> @Kira272921 Please do not post irrelevant links. This is not behaviour that belongs in our community. 🙂

😅 Sorry , will keep in mind next time

**Comment by avneesh0612** [comment]
Not exactly related to Open-source, but there are quite some advents going on-
- https://www.adventofcss.com/
- https://adventofjs.com/
- https://adventofcode.com/

**Comment by eddiejaoude** [comment]
How NOT to contribute to open source on GitHub - don't make these mistakes (5 tips)

 https://www.youtube.com/watch?v=ExFCwFPTsE0

**Comment by suitupstan** [comment]
Add an easter egg to eddiehub.org, when you type `ban`, a ban hammer gif appears, all the mod cards start dancing and a bg music starts playing. then a ban hammer starts to ban all elements from the page, finally all the mods ban themselves 😂 and the page reloads

**Reply by Vyvy-vi** [reply]
that can be awesome, though it might be hard to make that work with Kartra, unless we write external plugins

**Comment by suitupstan** [comment]
EddieHub Speed networking session v2

**Reply by Vyvy-vi** [reply]
That would be awesome!

**Reply by eddiejaoude** [reply]
I miss that! I want to do it again too!

**Comment by subhradwip01** [comment]
1.DevRel engineering
2. Product Management
3. Web3
